.github/workflows/openms_ci_matrix_full.yml

```diff
@@ -961,36 +963,64 @@ jobs:
         shell: bash
         run: echo "RUN_NAME=${{ github.event.pull_request && github.event.number || github.ref_name }}" >> $GITHUB_ENV
     
-      # NB we create the tag for the OpenMS repo next in a separate action.
+          # We created the draft release during deploy-installer step. Now we want to publish it.
+      - name: Publish OpenMS release
+        id: publish_release
+        if: inputs.do_release
+        uses: ncipollo/release-action@v1.16.0
+        with:
+          tag: ${{ github.ref_name }}
+          draft: false
+          allowUpdates: true
+          artifactErrorsFailBuild: true
+          makeLatest: ${{ inputs.mark_as_latest }}
+          omitBodyDuringUpdate: true
+          omitNameDuringUpdate: true
+          updateOnlyUnreleased: true
+
+
+          # NB we create the tag for the OpenMS repo next in a separate action.
       - id: bash_create_tags
         name: create tags for other repos
         shell: bash
         env: 
           GH_TOKEN: ${{ steps.app-token.outputs.token }}
         run: |
-          function createGitTag() {
+          function handleGitTag() {
```

Comment:
Maybe source or execute a script from the tools folder to keep the action clean once stable.

Reply:
Good idea. I'll add a note here so I remember after the next release cycle.